src/zope/schema/tests/test__bootstrapfields.py

```diff
@@ -958,11 +959,35 @@ def test_fromUnicode_miss(self):
         self.assertRaisesWrongType(txt.fromUnicode, txt._type, deadbeef)
 
     def test_fromUnicode_hit(self):
-
         deadbeef = u'DEADBEEF'
         txt = self._makeOne()
         self.assertEqual(txt.fromUnicode(deadbeef), deadbeef)
 
+    def test_normalization(self):
+        deadbeef = unicodedata.normalize('NFD', b'\xc3\x84\xc3\x96\xc3\x9c'.decode('utf-8'))
```

Comment:
Thank you! I find that this expresses the intent of the test case much more clearly than before, I can actually see how normalization changes the data.